src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -2,14 +2,13 @@
 // Licensed under the MIT License.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '127.0.0.1', but got 10.79.0.247.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```